app/build.gradle

```diff
@@ -16,6 +16,9 @@ android {
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
+    Properties properties = new Properties()
+    properties.load(project.rootProject.file("local.properties").newDataInputStream())
+    buildConfigField "String", "API_KEY", "\"${properties.getProperty("API_KEY")}\""
```

Comment:
Is this used anywhere?   If not in use, remove it.  

Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.

Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.